src/bootstrapGeneratedPackage.ts

```diff
@@ -1,14 +1,31 @@
-export type TBootstrapGeneratedPackage = (destinationDirectoryName: string) => any;
+export type TBootstrapGeneratedPackage = (destinationDirectoryName: string, language: string) => any;
 
-export default (exec: any): TBootstrapGeneratedPackage => (destinationDirectoryName: string) => {
-  const commandSequence = [
-    `cd ${destinationDirectoryName}`,
+interface ILanguageCommandSequences {
+  rust: string[];
+  typescript: string[];
+  [k: string]: string[];
+}
+
+const languageCommandSequences: ILanguageCommandSequences = {
+  rust: [
+    "cargo build",
+  ],
+  typescript: [
     "npm install",
     "npm run lint",
     "npm run build",
+  ],
+};
+
+export default (exec: any): TBootstrapGeneratedPackage => (destinationDirectoryName: string, language: string) => {
+
+  const commandSequence = [
+    `cd ${destinationDirectoryName}`,
+    ...languageCommandSequences[language],
   ].join(" && ");
 
-  return new Promise((resolve, reject) => exec(commandSequence, (err: Error, stdout: string) => {
+  return new Promise((resolve, reject) => exec(commandSequence, (err: Error, stdout: string, stdErr: string) => {
+    console.log(stdout, stdErr);
```

Comment:
cargo seems to actually use stdErr, need to make necessary provisions if this is the case.